cves/kernel/CVE-2014-9730.yml

```diff
@@ -326,8 +352,8 @@ vouch:
 
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of what your answer was.
-  answer:
-  note:
+  answer: true
+  note: "multiple mentions of the original reporter Carl H Lunde, thanking him"
```

Comment:
I think this question is more about people signing off on someone else's work, like for the implemented fix, there may be comments like "Acked by: ..." or "Signed off by: ..."